<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotControlWidget</class>
 <widget class="QWidget" name="PlotControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_0">
      <attribute name="title">
       <string>Margins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_0">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="xLowMarginLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X Low Margin</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="xHighMarginLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X High Margin</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="scaleTypeLabel">
           <property name="text">
            <string>Scale type</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="AnyComboBox" name="scaleTypeComboBox" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Legend</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_1">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="legendVisibilityCheckBox">
           <property name="text">
            <string>Show legend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="legendFrameVisibilityCheckBox">
           <property name="text">
            <string>Draw legend frame</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="legendFontSizeLabel">
             <property name="text">
              <string>Legend font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="legendFontSizeSpinBox">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Legend location</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LegendLayoutChoiceComboBox" name="legendLayoutComboBox" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Y</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_2">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Y tick font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="yAxisFontSizeSpinBox">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Y label font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="yLabelFontSizeSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="unitsVisibilityCheckBox">
           <property name="text">
            <string>Show units</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>X</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_3">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>X label</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LineEditWithUnicodeDialog" name="xLabelLineEdit" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>X label font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="xLabelFontSizeSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>X tick font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="xAxisFontSizeSpinBox">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>X tick rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="xTickRotationDoubleSpinBox">
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Time format</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="AnyComboBox" name="timeFormatComboBox" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Title</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_4">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TitleEditingWidget" name="titleEditingWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Axis rect</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_5">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AxisRectEditingWidget" name="axisRectEditingWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Grid</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_6">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GridEditingWidget" name="xGridWidget" native="true"/>
       </item>
       <item>
        <widget class="GridEditingWidget" name="yGridWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>SubGrid</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_7">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GridEditingWidget" name="xSubGridWidget" native="true"/>
       </item>
       <item>
        <widget class="GridEditingWidget" name="ySubGridWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Tick</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_8">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TickEditingWidget" name="tickEditingWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>SubTick</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_9">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TickEditingWidget" name="subTickEditingWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_10">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Event font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="eventFontSizeSpinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Event label rotation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="eventLabelRotationDoubleSpinBox">
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Arrowed text font size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="arrowedTextFontSizeSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_11">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorSetEditingWidget" name="colorSetEdititngWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Axes pref.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="FastenToolButton" name="fastenToolButton_12">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>X axis</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="AxisPrefEditingWidget" name="xAxisPrefWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Y axis</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="AxisPrefEditingWidget" name="yAxisPrefWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="defaultPushButton">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithUnicodeDialog</class>
   <extends>QWidget</extends>
   <header>commonWidgets/lineeditwithunicodedialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/anycombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FastenToolButton</class>
   <extends>QToolButton</extends>
   <header>controlWidgets/fastentoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GridEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/grideditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxisRectEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/axisrecteditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TickEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/tickeditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LegendLayoutChoiceComboBox</class>
   <extends>QWidget</extends>
   <header>commonWidgets/legendlayoutchoice/legendlayoutchoicecombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/titleeditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorSetEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/colorseteditingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxisPrefEditingWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/editingWidgets/axisprefeditingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
